src/bot.ts

```diff
@@ -272,15 +276,8 @@ export class DiscordBot {
         const result = await this.LookupRoom(guildId, channelId, event.sender);
         const chan = result.channel;
         const botUser = result.botUser;
-        let profile = null;
```

Comment:
We no longer need the profile for anything?

Reply:
That moved into matrixeventprocessor.ts `EventToEmbed` which was the only place where it was used. That was also an elegant fix for something with replies